app/models/task.rb

```diff
@@ -1,3 +1,4 @@
 class Task < ApplicationRecord
   belongs_to :list
+  belongs_to :label
```

Comment:
Maybe `has_many` association would be a better fit, task could have more than one label

Reply:
@nofaralfasi the labels are like the Google tasks categories? If so, I guess each task can have only one category, and in each category there's many tasks

Reply:
Yes, exactly. That was my intention.

Reply:
Maybe if I add `{:optional => true}` to the `belongs_to :label` it would be better?